src/uu/du/src/du.rs

```diff
@@ -61,15 +64,39 @@ impl Stat {
             path,
             is_dir: metadata.is_dir(),
             size: metadata.len(),
+
+            #[cfg(not(windows))]
```

Comment:
this is a lot of "#[cfg(not(windows))]"
would it be possible to find a better way?